custom_components/volkswagen_we_connect_id/sensor.py.save

```diff
@@ -0,0 +1,216 @@
+"""Sensor integration."""
```

Comment:
This file should be removed from the PR, looks like a backup you used during dev?